OPERATING.md

```diff
@@ -82,6 +82,11 @@ Should follow the following format:
     "temporal": ["<start-date>", "<end-date>"],
     "bounding_box": ["<bounding-box-as-comma-separated-LBRT>"],
     "include": "<filename-pattern>",
+
+    ## for inventory
+    "inventory_url": "s3://...",
+    "filename_regex": "<filename-regex>",
```

Comment:
The way this is written, it makes me believe the `filename_regex` key is a required parameter to make use of the inventory feature. However, it does not appear in the example step function input json file for icesat2-boreal -- you actually removed it, it had the value "^(.*).tif$". Is this now a default value somewhere for that parameter ?